config/sync/core.entity_form_display.node.directory_record.default.yml

```diff
@@ -108,6 +109,14 @@ content:
       size: 60
       placeholder: ''
     third_party_settings: {  }
+  field_footer_html:
```

Comment:
Okay, shouldn't field_footer_html and field_header_html be the same type? (same goes for state_directory_record).